airbyte-cdk/python/airbyte_cdk/sources/lcc/interpolation/interpolation.py

```diff
@@ -0,0 +1,14 @@
+#
+# Copyright (c) 2021 Airbyte, Inc., all rights reserved.
+#
+from abc import ABC, abstractmethod
+
+from airbyte_cdk.sources.lcc.types import Config
+
+
+class Interpolation(ABC):
```

Comment:
what is this for? I think this doesn't compile right? is it coming in a new PR?

Reply:
moved to another PR